app/src/main/java/com/example/lhi06/residue/findAlbum.java

```diff
@@ -0,0 +1,13 @@
+package com.example.lhi06.residue;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class findAlbum extends AppCompatActivity {
```

Comment:
Classes names are always camel case starting in upper case. And also activities normally have activity in the name. So `FindAlbumActivity` would be better